app/constants/storage.js

```diff
@@ -1,4 +1,5 @@
 export const LOCATION_DATA = 'LOCATION_DATA';
+export const INTERSECTIONS_DATA = 'INTERSECTIONS_DATA';
```

Comment:
Needs a better name. this is the list of already notified intersections, maybe `KNOWN_INTERSECTIONS`